.travis/install.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -ev
+
+if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+    hdiutil mount *firefox*.mac.dmg
+    sudo cp -R /Volumes/Nightly/FirefoxNightly.app /Applications
+    cd ~
+    hdiutil unmount /Volumes/Nightly/
```

Comment:
This way please: http://mxr.mozilla.org/comm-central/source/mozilla/testing/mozbase/mozinstall/mozinstall/mozinstall.py#240. There is no sudo necessary at all. The mounted DMG is in the user scope.

Also mozinstall works pretty fine for me via `mozinstall *firefox*.dmg` if there is only one single match in this folder. So please update it. Further also export the output path as an environment variable which can be re-used by the run section. That way we would not have to make a distinction between Linux and OS X.